modules/nw-sriov-configuring-device.adoc

```diff
@@ -89,3 +89,69 @@ A device can be used in either mode.
 $ oc create -f <filename> <1>
 ----
 <1>  Replace `<filename>` with the name of the file you created in the previous step.
+
+.NIC Partitioning
+
+In some case, you may want to split VFs of the same PF into multiple resource pools. For example, you want some of the VFs with the default driver loaded, while the rest with `vfio-pci` driver. To support such deployment, the `pfNames` selector can be used to specify a range of VFs for a pool in the below format:
+
+"<PFName>#<FirstVF>-<LastVF>"
+
+Where:
+
+. `<PFName>`   - is the PF interface name
+. `<FirstVF>`  - is the first VF index (0-based) that is included into the range
+. `<LastVF>`   - is the last VF index (0-based) that is included into the range
+
+Example:
+
+The selector for interface named netpf0 and VF 2 upto 7 (included 2 and 7) will look like:
+
+[source,yaml]
+pfNames: ["netpf0#2-7"]
+
+It is allowed to select the VFs from the same PF in different policy CRs, however you have to make sure:
+
+. The `numVfs` shall be identical among those policies, if they all select the same PF.
+. The VF index shall be in in range of `0` to `<numVfs>-1`. For example, if you have a policy with `numVfs` 8, then the `<FirstVF>` shall not be smaller than 0, while the `<FirstVF>` shall not larger than 7.
+. The VFs ranges in different policies shall not be overlapped.
+
+Example:
+
+The policy `policy-net-1` defined a resource pool `net-1` which contains the VF 0 to 7 of PF netpf0, with the default VF driver. The policy `policy-net-1-dpdk` defined another resource pool `net-1-dpdk` which contains the VF 8 to 15 of PF netpf0, with the vfio VF driver.
+
+Policy `policy-net-1`:
+
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-net-1
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: net-1
```

Comment:
the value for resourceName should not contain '-' character

Reply:
Fixed.